bin/katello-certs-check

```diff
@@ -249,6 +249,16 @@ function check-shortname () {
     fi
 }
 
+function check-ca-signing-algorithm () {
+    printf "Checking CA signing algorithm for sha1: "
+    CHECK=$(openssl x509 -noout -text -in $CA_BUNDLE_FILE | grep 'Signature Algorithm: sha1WithRSAEncryption')
+    if [[ $? == "0" ]]; then
+        error 4 "The server CA certificate has been signed with sha1 and will break installation."
```

Comment:
As with the maintain PR, I think the text should hint the user more to obtain a CA (!) that doesn't use SHA1

Reply:
"Please acquire a CA and server certificates signed with sha256 or greater." ?

Reply:
Something like that, yeah

Reply:
Don't forget to mention any intermediates. The whole chain needs to be free of SHA1.

Reply:
Check messaging now